### ISSUE TITLE
Evaluate independent predictions and generate heatmaps

### ISSUE THREAD

**Issue by sgould** [chengsoonong/beesee#18]
If possible produce heatmap for each part

**Comment by sgould** [comment]
Show heatmaps for 3 images in the training set and 3 images in the test set (6 x 7 = 42 images total). Indicate the peak (i.e., maximum) of heatmap and see if this corresponds to the predicted part location.
